libbeat/beat/beat.go

```diff
@@ -28,14 +28,13 @@ import (
 	"runtime"
 	"sync"
 
-	"github.com/satori/go.uuid"
-	"github.com/urso/ucfg"
-
 	"github.com/elastic/beats/libbeat/cfgfile"
+	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/filter"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/libbeat/publisher"
 	"github.com/elastic/beats/libbeat/service"
+	"github.com/satori/go.uuid"
```

Comment:
So far I assumed, if there is a new line inbetween, it should not get messed up?